### PR TITLE
Remove share prompt from classroom

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
# Issue

A prompt comes up which lets the student enter in their parents email address to show progress. There isn't supposed to be a prompt in the classroom version for parents emails.

# Fix

Check to make sure the user is not a student before showing modal. This preserves behavior for home users.

Tested fix manually on Firefox.

### CODE REVIEW

**Comment by AndrewJakubowicz** [comment]
I am unable to restart the travis builds. But the failures are due to apt-get.

**Comment by MarrriNee** [comment]
I’ve unsubscribed this one but still keep coming. How can I do? Could you check the issue or please just stop this subscription.
Thank you!!!

File: app/views/play/CampaignView.coffee [hunk 1]
```diff
@@ -431,7 +431,7 @@ module.exports = class CampaignView extends RootView
           @openModalView new CodePlayCreateAccountModal()
       else if me.get('name') and me.get('lastLevel') in ['forgetful-gemsmith', 'signs-and-portents', 'true-names'] and
       me.level() < 5 and not (me.get('ageRange') in ['18-24', '25-34', '35-44', '45-100']) and
-      not storage.load('sent-parent-email') and not me.isPremium()
+      not storage.load('sent-parent-email') and not me.isPremium() and not me.isStudent()
```
Comment (jmif) [inline app/views/play/CampaignView.coffee]: Do we care about teachers here or any other user types?  Should we whitelist to only home users?

**Comment by AndrewJakubowicz** [comment]
@MarrriNee I think you might be watching CodeCombat. You should be able to unwatch or unsubscribe yourself. I'm not sure I can do that.

File: app/views/play/CampaignView.coffee (hunk 1 above)
Comment (AndrewJakubowicz) [reply app/views/play/CampaignView.coffee]: This is a pretty good point. I'll add a `not me.isTeacher()` and manually test.

Comment (AndrewJakubowicz) [reply app/views/play/CampaignView.coffee]: @jmif Made this logic include teachers (and did a De Morgan transformation as I think it is easier to understand).
Does this look good to you?

**Pull request merged**